src/main/kotlin/com/cognifide/gradle/aem/base/api/AemConfig.kt

```diff
@@ -41,13 +41,13 @@ open class AemConfig(project: Project) : Serializable {
      * Determines current environment to be used in deployment.
      */
     @Input
-    val deployEnvironment : String = propParser.string("aem.env", { System.getenv("AEM_ENV") ?: "local" })
+    val deployEnvironment: String = propParser.string("aem.env", { System.getenv("AEM_ENV") ?: "local" })
 
     /**
      * Determines instances involved in CRX package deployment.
      */
     @Input
-    var deployInstanceName : String = propParser.string("aem.deploy.instance.name", "*-$deployEnvironment")
+    var deployInstanceName: String = propParser.string("aem.deploy.instance.name", "$deployEnvironment-*")
```

Comment:
what does "-*" do?

Reply:
Instance name is combination of $env-$type where env could be local int stg and so on and $type could be author or publish. local-* means "deploy crx packages to all local instances of any type"